go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/openshift/cluster-network-operator
 
-go 1.15
+go 1.16
```

Comment:
This requires a change to Dockerfile too. (Oddly, Dockerfile.rhel7 already says 1.16)

Reply:
done